content/knowledge_base/glossary.md

```diff
@@ -0,0 +1,77 @@
+# Glossary
+
+## Add a subheading
+
+### Ancestor Set
+
+TODO: add
+
+### Chit
+
+In Avalanche context: a transaction receives a chit (boolean value) when the majority (quorum) of queried nodes respond with an approval for that transaction.
```

Comment:
If some terms of the glossary are within a given context, I would make them subsections of a section named like that context. For example here, I would have this entry as part of 

```markdown
# Glossary

## Avalanche

[Something about what is avalanche etc.]

### Chit

[What is a Chit in Avalanche]
```

WDYT?

Reply:
This makes sense yes; Since we are having multiple topics with definitions
Thanks!

Reply:
Added the logical separation in sections https://github.com/optakt/optakt.github.io/pull/13/commits/47d47376667d886f5b56d7bfcd03e1f14a6086b0#diff-55a59bbede67c988e24e88ffb8a7b56d20454b9ab95a3fd8567e790585338dfbR3-R4
Do they seem reasonable to you?

Reply:
Yeah I like it :) I think it will be much clearer this way! Thanks.